### DISCUSSION TITLE
more info about `autload = yes ` in uosc.conf

### DISCUSSION THREAD

**Discussion by OmiChrome** [tomasklaen/uosc#507] [General]
since the original 
[autoload](https://github.com/mpv-player/mpv/blob/master/TOOLS/lua/autoload.lua) is not working , 
the issue was it just exit the player when it adds new file  , still it persits that bug 

so i think to switch to uosc `autoload`
https://github.com/tomasklaen/uosc/blob/ec52252380f896ca709216307e3bf021fbee914b/script-opts/uosc.conf#L144

but its not **working**  as `autoload` has to be 
`uosc.conf`
![image](https://user-images.githubusercontent.com/108952834/232251636-d637b274-0f85-4280-9c7a-7ab8b2b4d8e0.png)
`mpv.conf`
![image](https://user-images.githubusercontent.com/108952834/232251648-a8a28d95-2cdb-46df-9032-74ff87d39ec1.png)

every thiing set up as instructions  said , 

but it din't worked it work as i was expected to,  

## wrong !     by using `uosc.conf`
![image](https://user-images.githubusercontent.com/108952834/232251697-efa45d47-e8d3-4339-95e6-795e5393caaf.png)

## right!    by droping whole folder to mpv 
![image](https://user-images.githubusercontent.com/108952834/232251724-6135b7de-174d-44a6-8318-a290f809bc61.png)

**Comment by OmiChrome** [comment]
this view is called contact-sheet [playlist-view](https://github.com/occivink/mpv-gallery-view)

**Comment by po5** [comment]
How you aren't already banned from everyone else's repos is beyond me.

**Comment by christoph-heinrich** [comment]
https://github.com/tomasklaen/uosc/blob/ec52252380f896ca709216307e3bf021fbee914b/script-opts/uosc.conf#L142

autoload in uosc doesn't work like autoload.lua. It doesn't add other files from the same folder to the playlist beforehand, instead it looks up the next file in the folder when your current file ends. Have you tried letting the file play to the end?

Also what do you mean autoload.lua exits mpv? It shouldn't be able to do that.

**Reply by OmiChrome** [reply]
> play to the end?
yes i have used that way too ! 

my question is now , can it work like autoload.lua , like add to playlist , ?

**Reply by hooke007** [reply]
No

**Comment by hooke007** [comment]
> since the original autoload is not working

autoload.lua seem to be the most popular external scripts for mpv, never heard that someone reporting it doesn't work.

> so i think to switch to uosc autoload

Just as the instruction of comments of uosc's autoload, it doen't add any items into the playlist. I don't think uosc's autoload is the alternative feature of autoload.lua
playlist-view.lua do no help here. Because you don't create a playlist with multiple items.

**Reply by OmiChrome** [reply]
> autoload.lua seem to be the most popular external scripts for mpv, never heard that someone reporting it doesn't work.
> 
yes it `actually not working`
 

https://user-images.githubusercontent.com/108952834/232289672-06a10f73-df0f-411a-a5d2-1f4a82ffd3e0.mov

**log** with autoload.lua
[MPV.log](https://github.com/tomasklaen/uosc/files/11241777/MPV.log)

**log** without autoload.lua 
[MPV.log](https://github.com/tomasklaen/uosc/files/11241779/MPV.log)


tldr : autoload is making `mpv` exit when it starts 

using Default [autoload](https://github.com/mpv-player/mpv/blob/master/TOOLS/lua/autoload.lua)

**Reply by hooke007** [reply]
You should figure it out or report to mpv.
